### ISSUE TITLE
Follow up #1200

### ISSUE THREAD

**Issue by FireBladeNooT** [pymedusa/Medusa#1232]
Following this bug: Anime episode not recognized #1200

Hi again,
I'm sorry but issue is not solved,
2016-10-07 19:08:43 INFO Thread-24 :: [2697d37] Not enough information to determine what episode this is. Quitting post-processing
2016-10-07 19:08:43 DEBUG Thread-24 :: [2697d37] Unable to match Finish [Miracle] Dragon Ball Super Episode 55 [VOSTFR][720p].mp4 to a show in your database. Parser result: title: Finish, release_group: Miracle, episode_title: Dragon Ball Super, episode: [55], subtitle_language: fr, screen_size: 720p, container: mp4, mimetype: video/mp4, type: episode, absolute_episode: [], quality: Unknown, season: None
2016-10-07 19:08:42 DEBUG Thread-24 :: [2697d37] Analyzing name Finish [Miracle] Dragon Ball Super Episode 55 [VOSTFR][720p].mp4

I've version 2016-10-05 - 0.1.6

The parser thinks "series name" is "episode title" :(

**Comment by ratoaq2** [comment]
What's the video name?
`Finish [Miracle] Dragon Ball Super Episode 55 [VOSTFR][720p].mp4` ?

Why there's Finish at the beginning and Miracle inside square brackets?
The show name is Dragon Ball Super Episode 55, correct?

**Comment by medariox** [comment]
ping @FireBladeNooT

**Comment by FireBladeNooT** [comment]
Sorry, for not reply,
the video name is [Miracle] Dragon Ball Super Episode 55 [VOSTFR][720p].mp4.
Finish is the folder name.

**Comment by FireBladeNooT** [comment]
2016-10-15 12:13:14 WARNING  Thread-16 :: [2697d37] [/volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4 : Processing failed: ]
2016-10-15 12:13:14 WARNING  Thread-16 :: [2697d37] Problem(s) during processing, failed the following files/folders:  
2016-10-15 12:13:14 WARNING  Thread-16 :: [2697d37] Processing failed for /volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4: 
2016-10-15 12:13:14 INFO     Thread-16 :: [2697d37] Not enough information to determine what episode this is. Quitting post-processing
2016-10-15 12:13:14 DEBUG    Thread-16 :: [2697d37] Unable to match Finish [Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4 to a show in your database. Parser result: title: Finish, release_group: Miracle, episode_title: Dragon Ball Super, episode: [61], subtitle_language: fr, screen_size: 720p, container: [u'MP4', u'mp4'], mimetype: video/mp4, type: episode, absolute_episode: [], quality: Unknown, season: None
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Analyzing name Finish [Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parse result(release_group): Miracle
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37]  or Parse result(air_date): None
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parse result(episode_numbers): [61]
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parse result(season_number): None
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parse result(series_name): Dragon Ball Super
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parse result not sufficient (all following have to be set). will not save release name
2016-10-15 12:13:13 DEBUG    Thread-16 :: [2697d37] Parsed /volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4 into release_group: Miracle, title: Dragon Ball Super, episode: [61], subtitle_language: fr, screen_size: 720p, container: [u'MP4', u'mp4'], mimetype: video/mp4, type: episode, absolute_episode: [], quality: 720p HDTV, season: None
2016-10-15 12:13:12 DEBUG    Thread-16 :: [2697d37] Analyzing name /volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4
2016-10-15 12:13:12 DEBUG    Thread-16 :: [2697d37] Unable to match Finish to a show in your database. Parser result: title: Finish, type: episode, absolute_episode: [], quality: Unknown, episode: [], season: None
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Analyzing name Finish
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parse result(release_group): Miracle
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37]  or Parse result(air_date): None
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parse result(episode_numbers): [61]
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parse result(season_number): None
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parse result(series_name): Dragon Ball Super
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parse result not sufficient (all following have to be set). will not save release name
2016-10-15 12:13:11 DEBUG    Thread-16 :: [2697d37] Parsed [Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4 into release_group: Miracle, title: Dragon Ball Super, episode: [61], subtitle_language: fr, screen_size: 720p, container: [u'MP4', u'mp4'], mimetype: video/mp4, type: episode, absolute_episode: [], quality: 720p HDTV, season: None
2016-10-15 12:13:10 DEBUG    Thread-16 :: [2697d37] Analyzing name [Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4
2016-10-15 12:13:10 INFO     Thread-16 :: [2697d37] Processing /volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4 (Torrent)
2016-10-15 12:13:04 DEBUG    Thread-16 :: [2697d37] Trying to find download on theTVDB
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] PostProcessing VideoInRar: 
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] PostProcessing RarContent: 
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] PostProcessing VideoFiles: ['[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4']
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] PostProcessing Files: ['[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4.mp4']
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] PostProcessing Dirs: ['@eaDir']
2016-10-15 12:13:03 INFO     Thread-16 :: [2697d37] PostProcessing Path: /volume1/Download/Finish
2016-10-15 12:13:03 DEBUG    Thread-16 :: [2697d37] Processing folder /volume1/Download/Finish
2016-10-15 12:10:49 DEBUG    Thread-16 :: [2697d37] Looking up XEM scene mapping for show 295068 on theTVDB

**Comment by OmgImAlexis** [comment]
@FireBladeNooT [just a tip](https://help.github.com/articles/creating-and-highlighting-code-blocks).

**Comment by FireBladeNooT** [comment]
We can see with the log that first it analyze "[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4", then "Finish" and finally "Finish [Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4"

**Comment by FireBladeNooT** [comment]
```
2016-10-15 12:25:55 DEBUG    Thread-16 :: [2697d37] Folder name (Finish) appears to be a valid release name. Using it.
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Found release name [Miracle] Dragon Ball Super 61 [VOSTFR][720p]
2016-10-15 12:25:53 INFO     Thread-16 :: [2697d37] Trying to clean any empty folders under /volume1/video/TV show/Dragon Ball Super/Season 04
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] No associated files for /volume1/video/TV show/Dragon Ball Super/Season 04/Dragon Ball Super - S04E15 - Les ambitions de Zamasu - Le plan d'éradication des Humains.mp4 were found during this pass
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Trying to determine free space on destination drive
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] File /volume1/video/TV show/Dragon Ball Super/Season 04/Dragon Ball Super - S04E15 - Les ambitions de Zamasu - Le plan d'éradication des Humains.mp4 doesn't exist so there's no worries about replacing it
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] This episode is a priority download: False
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Quality of the episode we are processing: 720p HDTV
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Snatch history had a quality in it, using that: 720p HDTV
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Retrieving episode object for 4x15
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Parsed [Miracle] Dragon Ball Super 61 [VOSTFR][720p].mp4 into release_group: Miracle, title: Dragon Ball Super, episode: [15], absolute_episode: [61], subtitle_language: fr, screen_size: 720p, container: mp4, mimetype: video/mp4, type: episode, quality: 720p HDTV, season: 4
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] Scene show Dragon Ball Super using indexer for absolute 61: S4E[15]
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] 295068: Setter sets location to /volume1/video/TV show/Dragon Ball Super/Season 04/Dragon Ball Super - S04E15 - Les ambitions de Zamasu - Le plan d'éradication des Humains.mp4
2016-10-15 12:25:53 DEBUG    Thread-16 :: [2697d37] 295068: Found episode by absolute number 61 which is Dragon Ball Super S04E15
2016-10-15 12:25:52 DEBUG    Thread-16 :: [2697d37] Scene show Dragon Ball Super is anime
2016-10-15 12:25:52 DEBUG    Thread-16 :: [2697d37] Analyzing name [Miracle] Dragon Ball Super 61 [VOSTFR][720p].mp4
2016-10-15 12:25:51 INFO     Thread-16 :: [2697d37] Processing /volume1/Download/Finish/[Miracle] Dragon Ball Super 61 [VOSTFR][720p].mp4 (Torrent)
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] Trying to find download on theTVDB
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] PostProcessing VideoInRar: 
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] PostProcessing RarContent: 
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] PostProcessing VideoFiles: ['[Miracle] Dragon Ball Super 61 [VOSTFR][720p].mp4']
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] PostProcessing Files: ['[Miracle] Dragon Ball Super 61 [VOSTFR][720p].mp4']
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] PostProcessing Dirs: ['@eaDir']
2016-10-15 12:25:45 INFO     Thread-16 :: [2697d37] PostProcessing Path: /volume1/Download/Finish
2016-10-15 12:25:45 DEBUG    Thread-16 :: [2697d37] Processing folder /volume1/Download/Finish

**Comment by FireBladeNooT** [comment]
Works fine without Episode in the name

**Comment by medariox** [comment]
Guessit still doesn't parse it as an absolute episode.

**Comment by OmgImAlexis** [comment]
@medariox looks like this is actually a bug since guessit seems fine manually running this.

Running guessit 2.0.5.

```
For: /volume1/Download/Finish/[Miracle] Dragon Ball Super Episode 61 [VOSTFR][720p].mp4
GuessIt found: {
    "release_group": "Miracle", 
    "title": "Dragon Ball Super", 
    "episode": 61, 
    "subtitle_language": "French", 
    "screen_size": "720p", 
    "container": "mp4", 
    "mimetype": "video/mp4", 
    "type": "episode"
}
```

**Comment by medariox** [comment]
@OmgImAlexis
Nope, episode should be absolute_episode. Also make sure to use our version of guessit, as you might get different results otherwise (the easiest way is to use report_guessit.py).

**Comment by FireBladeNooT** [comment]
@medariox @OmgImAlexis 
It is normal that it search "Finish" and "Finish FileName" ?

**Comment by FireBladeNooT** [comment]
Branch: master
Commit: 2697d373701d7bea6dcfe6a651e3d2ff0a23df80

**Comment by medariox** [comment]
@FireBladeNooT 
Yes, it is. This behavior will be improved in the next version, I'm currently working on it.

**Comment by ratoaq2** [comment]
Fixed in develop by #1271
